docs/unified-test-documentation/dasharo-compatibility/309-qubesos-stable-support.md

```diff
@@ -0,0 +1,63 @@
+# Dasharo Compatibility: QubesOS support
```

Comment:
@PrzemyslawBanasiak13 This file should be added to `mkdocs` file. Without this documentation will be unavailable. Also please improve test matrixes for NC laptops.